utils/macos/build_app.sh

```diff
@@ -77,13 +77,15 @@ if [ ! -d "$SDK_DIRECTORY" ]; then
    fi
 fi
 
-REVISION=`bzr revno $SOURCE_DIR`
+REVISION=`git rev-parse --short=5 HEAD`[`git --git-dir=$SOURCE_DIR/.git rev-parse --abbrev-ref HEAD`]
```

Comment:
Not sure why you'd want the branch in there. The SHA already is unique and identifiable. Also, your 'master' branch could be my 'awesome' branch, git does not guarantee uniqueness of branch names across checkouts. I suggest dropping the branch name.

Reply:
My suggestion remains `"r$(git rev-list --count HEAD)_$(git rev-parse --short HEAD)"`. Having the number of commits in there makes lexicographical sorting of consecutive builds possible, the SHA though is the real identifier. I strongly suggest sticking with `short=7`, so that the numbers are similar than on github. Which is very useful for recognition.

Reply:
Although it is meaningless, the branch name will help us when low-tech people make bug reports on what they have been testing. If they have been testing a branch, we will automatically know that fro the version info.

+1 on being similar to GitHub

Reply:
Okay, good arguments. I concur.